hashintel/hash
# Add button for converting entity types to link types

## 🌟 What is the purpose of this PR?



This PR adds a button to convert Entity Types to Link Types.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203179076056209/1204600153441697/f) _(internal)_

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:












### 📜 Does this require a change to the docs?




The changes in this PR:






### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:






## 🐾 Next steps


Add a button to convert a Link Type to an Entity Type (further discussion is required).


## ❓ How to test this?



1.  Create an Entity Type;
2.  Click on the button to convert it to a Link Type;
3.  Make sure the Entity Type was converted to a Link Type.

## 📹 Demo



https://github.com/hashintel/hash/assets/37453800/b4d05a7d-8558-444c-a33c-0e5a48d79706
